### PR TITLE
chore: Test empty output

### PR DESCRIPTION
**Pull request by dark0dave**
Ensuring json returns empty array and text stdout is empty when no issues are found

Signed-off-by: dark0dave <dark0dave@mykolab.com>

### CODE REVIEW

File: cmd/kubent/main_test.go [hunk 1]
```diff
@@ -99,18 +99,26 @@ func TestStoreCollectorError(t *testing.T) {
...
-		{"version long flag set", []string{"--version"}, 0},
+		{"success", []string{clusterFlagDisabled, helm2FlagDisabled, helm3FlagDisabled}, 0, ""},
+		{"errorBadFlag", []string{"-c=not-boolean"}, 2, ""},
+		{"successFound", []string{"-o=json", clusterFlagDisabled, helm2FlagDisabled, helm3FlagDisabled, "-f=" + filepath.Join(FIXTURES_DIR, "deployment-v1beta1.yaml")}, 0, string(expectedJsonOutput)},
+		{"exitErrorFlagNone", []string{clusterFlagDisabled, helm2FlagDisabled, helm3FlagDisabled, "-e"}, 0, ""},
+		{"exitErrorFlagFound", []string{clusterFlagDisabled, helm2FlagDisabled, helm3FlagDisabled, "-e", "-f=" + filepath.Join(FIXTURES_DIR, "deployment-v1beta1.yaml")}, 200, ""},
+		{"version short flag set", []string{"-v"}, 0, ""},
+		{"version long flag set", []string{"--version"}, 0, ""},
+		{"empty text output", []string{}, 0, ""},
+		{"empty json output", []string{"-o=json"}, 0, `[]
```
Comment (stepanstipl) [inline cmd/kubent/main_test.go:120]: Wouldn't `"[]\n"` work here? Assuming this is about the new line...

**Review by stepanstipl — COMMENTED** [review]
Just a minor thingie with the line split, looks good otherwise 👍 

I see you dealt with the Sonar Cloud slave 🤣

File: cmd/kubent/main_test.go (hunk 1 above)
Comment (dark0dave) [reply cmd/kubent/main_test.go:120]: yes can do

**Review by stepanstipl — APPROVED** [review]
👍 🙏

**Pull request merged**